rtv/terminal.py

```diff
@@ -438,6 +439,20 @@ def open_editor(self, data=''):
             else:
                 _logger.info('File deleted: %s', filepath)
 
+    def open_urlview(self, data):
+        urlview = os.getenv('URLVIEWER') or os.getenv('RTV_URLVIEWER') \
+            or 'urlview'
+        try:
+            with self.suspend():
+                p = subprocess.Popen([urlview],
+                                     stdin=subprocess.PIPE)
+                try:
+                    p.communicate(input=data)
+                except KeyboardInterrupt:
+                    p.terminate()
+        except OSError:
+            self.show_notification('Could not open urls with urlview')
```

Comment:
The notification should show the command defined by the environment variable if applicable